packages/synapse-react-client/src/components/QueryWrapper/QueryWrapper.tsx

```diff
@@ -52,6 +52,11 @@ export type QueryWrapperProps = React.PropsWithChildren<{
   isRowSelectionUIFloating?: boolean
   isInfinite?: boolean
   combineRangeFacetConfig?: CombineRangeFacetConfig
+  /** If provided, will use the value in this column instead of the rowID for the access column, download column, etc */
```

Comment:
Move these to the QueryWrapper, and set atoms instead of passing plumbing them through the hierarchy.